lib/services/events_S/event_service.dart

```diff
@@ -109,6 +129,21 @@ class Events {
     }
   }
 
+  Future<void> declineEventInvitation(Event event) async {
+    final DocumentReference docRef = FirebaseFirestore.instance.collection('events').doc(event.eventID);
+
+    try {
+      final currentUser = FirebaseAuth.instance.currentUser;
+      if (currentUser != null) {
+        await docRef.update({
+          'invited_users': FieldValue.arrayRemove([currentUser.uid])
```

Comment:
the field I used was just called "invited" not invited_users

Reply:
That's odd...it worked on my end...did it work on yours?

Reply:
no it says no new event invites on my side

Reply:
Ohhh...somehow only the first document has "invited_users".  Can I remove that and change the code to "invited"?

Reply:
yes please do

Reply:
I changed it to 'invite'

Reply:
I noticed on this document: AlV61ZYE6FARUAHT3xv1 taht there is both, could explain why it worked on Franco's side and not on the others. Nevertheless I see it has been updated.